storage/shard.go

```diff
@@ -1699,12 +1699,19 @@ func (s *dbShard) Flush(
 
 	var multiErr xerrors.MultiError
 	tmpCtx := context.NewContext()
+
+	var (
+		// Debug
+		numBlockDoesNotExist  = 0
+		numStreamDoesNotExist = 0
```

Comment:
Do you still need these post-debug?

Reply:
good to have for debugging and shouldn't really cost us anything

Reply:
spoke in person, gonna keep em but return as a result

Reply:
done, emitting metrics in the shard now